main.go

```diff
@@ -0,0 +1,123 @@
+package authenticatedblockstore
+
+import (
+	"crypto/sha256"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"math/rand"
+	"os"
+	"strings"
+	"time"
+)
+
+// This is a test file for testing the package.
+// It is not intended to be part of the package.
```

Comment:
If it is not intended to be part of this package, why is it? Needs some explanation, or to be removed from the package

Reply:
Yes, the file should not be part of the commit. I have removed it.